common/app/model/dotcomrendering/MostPopular.scala

```diff
@@ -187,8 +186,24 @@ object MostPopularGeoResponse {
 // MostPopularNx2 was introduced to replace the less flexible [common] MostPopular
 // which is heavily relying on pressed.PressedContent
 // because we want to be able to create MostPopularNx2 from trails coming from the DeeplyReadAgent
-case class MostPopularNx2(heading: String, section: String, trails: Seq[OnwardItemNx2])
+case class MostPopularNx2(heading: String, section: String, trails: Seq[OnwardItem])
```

Comment:
It'd be great to consolidate this class one day too - might be harder given the comment above.

Reply:
We determined this is a good TBD